static/src/javascripts/projects/common/modules/ui/cmp-ui.js

```diff
@@ -1,24 +1,37 @@
 // @flow
 import { getCookie } from 'lib/cookies';
 
-// TODO: this should be derived from config
-const CMP_DOMAIN = 'https://manage.theguardian.com';
+// TODO: this should be derived from config or imported from new lib
+const CMP_DOMAIN = 'https://manage.thegulocal.com';
```

Comment:
You forgot to change this back to the prod domain.
```suggestion
const CMP_DOMAIN = 'https://manage.theguardian..com';
```